lib/dai-plugin-mcd/test/helpers.js

```diff
@@ -63,7 +63,7 @@ export async function setDebtCeiling(maker, amount, ilk) {
       ilk && stringToBytes(ilk),
       stringToBytes(ilk ? 'line' : 'Line'),
       ethAbi.encodeParameter('uint', amount.toFixed('rad')),
-      { dsProxy: scs.getContractAddress('MCD_MOM') }
+      { dsProxy: scs.getContractAddress('MCD_MOM'), gasLimit: 4000000 }
```

Comment:
is this necessary? I tried deleting it and tests still passed

Reply:
I think it's not necessary anymore now that I added the fallback gas limit to the estimator service. Good catch 👍